contributing.md

```diff
@@ -1,6 +1,23 @@
 # Contributors Guidelines to PyDP
 
-## About PyDP
+## Getting Started
+
+### Learn Git and Github
+
+All our development is done using Git and Github. If you're not too familiar with Git and Github, start by reviewing this guide. <https://guides.github.com/activities/hello-world/>
```

Comment:
`Git on Github` sounds to be correct usage?

Reply:
okey

Reply:
"git and Github" seems to be the correct usage @chinmayshah99